lib/shopify_api/resources/variant.rb

```diff
@@ -26,10 +26,9 @@ def old_inventory_quantity=(new_value)
       super
     end
 
-    def serializable_hash(options = {})
-      super(options).tap do |resource|
-        (resource['variant'] || resource).except!('inventory_quantity', 'old_inventory_quantity')
-      end
+    def save
+      attributes.except!('inventory_quantity', 'old_inventory_quantity')
+      super
```

Comment:
This is great, though we are still dropping the fields. Would it make sense to restore the original attributes in an `ensure` block?

Reply:
Resolved offline, original attributes are still present after the save!